### PR TITLE
Refund transaction support

### PR DESCRIPTION
**Pull request by architect0317**
Added support for refund transaction calls to Authorize.Net API

### CODE REVIEW

**Comment by architect0317** [comment]
I'm very new to this process, so if I'm doing anything incorrectly on the PR, let me know. I had some lingering setting issues in my IntelliJ code style that I believe are now corrected so I believe we are using the correct code style standards.

**Comment by kayladnls** [comment]
Thanks for the PR! There are a few styling issues (i've made inline comments on those). 

Other than that, I would just need some tests to cover this code, and I'll be happy to merge it in.

File: src/AIMGateway.php [hunk 1]
```diff
@@ -59,21 +59,36 @@ public function setDeveloperMode($value)
 
     public function authorize(array $parameters = array())
     {
-        return $this->createRequest('\Omnipay\AuthorizeNet\Message\AIMAuthorizeRequest', $parameters);
+        return $this->createRequest(
```
Comment (kayladnls) [inline src/AIMGateway.php]: These lines don't need to wrap.

**Comment by architect0317** [comment]
Thanks! I ran PHP Code Sniffer against it and it wrapped a couple of lines funny, so I'll correct those and get the test coverage done as well. What is the official code style standard we should be following?

**Comment by architect0317** [comment]
We added the unit tests over the weekend and I fixed my code style settings so it won't auto-wrap function calls any more.

**Comment by kayladnls** [comment]
+1 Looks great, thank you!

**Pull request merged**